local-pv-provisioner/controllers/device_detector.go

```diff
@@ -111,3 +128,55 @@ func (dd *DeviceDetector) do() error {
 
 	return nil
 }
+
+func (dd *DeviceDetector) pvName(devPath string) string {
```

Comment:
The receiver methods of DeviceDetector are just moved from `persisntet_volume.go`.